bc-backend/src/main/java/org/jboss/da/bc/model/backend/ProjectDetail.java

```diff
@@ -65,7 +65,11 @@
 
     @Getter
     @Setter
-    private boolean useExistingBc = false; // required, default true when bcExists
+    private Integer useExistingBc; // required, default non-null when bcExists
+
+    @Getter
+    @Setter
+    private List<Integer> existingBCs;
```

Comment:
You can delete the field ``bcID`` as it is replaced by this list ``existingBCs``.